docker/Dockerfile-Ubuntu-16.04

```diff
@@ -36,35 +38,36 @@ RUN wget http://archive.ubuntu.com/ubuntu/pool/universe/a/astyle/astyle_3.0.1-1u
     && dpkg -i astyle_3.0.1-1ubuntu1_amd64.deb \
     && rm astyle_3.0.1-1ubuntu1_amd64.deb
 
-RUN pip install grpcio grpcio-tools
-
-# Install gRPC according to https://github.com/grpc/grpc/blob/v1.6.x/INSTALL.md
-# And to fix build error https://github.com/grpc/grpc/issues/12316
-# Install gRPC and protobuf into /opt in order not to mess up the host system.
-# Also, we need to patch the Makefile because it install the libraries with
-# wrong versions.
-
-RUN mkdir /opt/grpc && chown $USER:$USER /opt/grpc
-RUN mkdir /opt/protobuf && chown $USER:$USER /opt/protobuf
-
-ADD grpc/0001-Makefile-fix-wrong-C-version-suffix.patch 0001-Makefile-fix-wrong-C-version-suffix.patch
-
-RUN git clone -b v1.7.x https://github.com/grpc/grpc \
-    && cd grpc \
-    && git submodule update --init \
-    && git apply ../0001-Makefile-fix-wrong-C-version-suffix.patch \
-    && make -j8 HAS_SYSTEM_PROTOBUF=false \
-    && make prefix=/opt/grpc install \
-    && cd third_party/protobuf \
-    && make clean \
-    && ./configure --prefix=/opt/protobuf \
-    && make -j8 \
-    && make install \
-    && cd ../../.. \
-    && rm -rf grpc
-
-ENV GRPC_DIR=/opt/grpc
-ENV PROTOBUF_DIR=/opt/protobuf
+# Everything gRPC below is commented out because we'll build gRPC as part of the main build.
```

Comment:
I'd say we remove the comments :-). IMO the next best solution after building grpc in our build system is to use a package manager.

Reply:
Tend to agree.  Change comment above to some unspecific comment about gRPC build being handled separately.